src/runtime/database.cpp

```diff
@@ -995,7 +1015,11 @@ static std::string format_time(int64_t ns) {
   return buf;
 }
 
-static JobReflection find_one(Database *db, sqlite3_stmt *query, bool verbose) {
+std::string Time::as_string() const {
+    return format_time(t);
+}
```

Comment:
We could have just removed format_time and put it in here.

Reply:
format_time() is also used in `  desc.wake_start     = format_time(sqlite3_column_int64(query, 10));` in database.cpp. Do we want `JobReflection.wake_start` to be Time as well?

Reply:
Done in #858